common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -177,9 +177,16 @@ def handle_ajax(self, dispatch, data):  # TODO: bounds checking
         raise NotFoundError('Unexpected dispatch type')
 
     def student_view(self, context):
+        display_items = self.get_display_items()
+
         # If we're rendering this sequence, but no position is set yet,
+        # or exceeds the length of the displayable items,
         # default the position to the first element
-        if self.position is None:
+        if context.get('requested_child') == 'first':
+            self.position = 1
+        elif context.get('requested_child') == 'last':
+            self.position = len(display_items) or None
+        elif self.position is None or self.position > len(display_items):
```

Comment:
Why does specifying a `position` greater than `len(display_items)` return `1`, and not `len(display_items)`?